### ISSUE TITLE
namespaced secret syncer

### ISSUE THREAD

**Issue by kfox1111** [appscode/kubed#383]
I'm interested in trying out kubed in a multitenant environment. Rather then deploy it cluster wide, I'd like to deploy it with just enough permissions to sync from one namespace to another.

The current chart does not seem to have enough functionality to do so.

**Comment by tebo-ernstberger** [comment]
For that particular need I would recommend https://github.com/mittwald/kubernetes-replicator which requires the creation of an empty configmap/secret that will receive the data of another configmap/secret. 
In a multi-tenant environment, as long as the user can create/update both resources (RBAC) then you would get the safety you need.

If you really want to use kubed, you might get away with it by implementing some OPA policies. But that also depends on how you handle multi tenancy.

**Comment by kfox1111** [comment]
kubernetes-replicator has the same issue. cert-manager doesn't allow you to create secrets with annotations.

**Comment by tebo-ernstberger** [comment]
Yes it does, just create an empty secret first with the annotations.

**Comment by kfox1111** [comment]
yes/no....

I want to manage everything with a helm chart. I can't lay down the secret with helm, as it will think it owns everything and will revert things on upgrade. the secret shouldn't be managed by both cert-manager and helm. :/
